news/2021-02-01-mappers-report-january.md

```diff
@@ -0,0 +1,133 @@
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+This month's issue of the Mappers' Report contains following topics:
+
+- **[Ranking Criteria Changes](#rc-changes)**
+- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
+- **[Mappers’ Showcase](#mappers-showcase)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Around the Community](#around-community)**
+- **[Conclusion](#conclusion)**
+
+## <a id="rc-changes"></a>Ranking Criteria Changes
+
+The ranking criteria continues to change, even in 2021. Here we’ve gathered all changes which happened so far.
+
+### osu!catch
+
+- **[Allow dashes over 250ms before higher-snapped hyperdashes in Platters](https://osu.ppy.sh/community/forums/topics/1188695):** This change didn’t add or remove any rules or guidelines. It split existing rules in Platter and Rain difficulties into two separate rules for simplification purposes.
+
+## Open Ranking Criteria Proposals 
+
+All modes:
+
+- **[Beatmaps of explicit audio tracks need to have "Explicit Content" setting enabled](https://osu.ppy.sh/community/forums/topics/1218048)**
+- **[Adjusting the song cut guideline](https://osu.ppy.sh/community/forums/topics/1231363)**
+
+For osu!:
+
+- **[Let spinners start/end less than 1/16 from the prior/following object](https://osu.ppy.sh/community/forums/topics/1214161)**
+
+For osu!mania:
+
+- **[Remove Guidelines regarding SVs](https://osu.ppy.sh/community/forums/topics/1227918)**
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+### BN Additions
+
+- **osu!:** [rosario wknd](https://osu.ppy.sh/users/6341518) & [Xahlt](https://osu.ppy.sh/users/13466351)
+- **osu!catch:** [Dako](https://osu.ppy.sh/users/11081858) & [Myxo](https://osu.ppy.sh/users/2202645)
+- **osu!mania:** [guden](https://osu.ppy.sh/users/11626065) & [deta5859](https://osu.ppy.sh/users/15275662)
+
+### BN Resignations
+
+- **osu!:** [Okoratu](https://osu.ppy.sh/users/1623405)
+- **osu!taiko:** [Greenshell](https://osu.ppy.sh/users/8693851)
+- **osu!catch:** [JierYagtama](https://osu.ppy.sh/users/7483452) & [-Joni-](https://osu.ppy.sh/users/9988837)
+- **osu!mania:** [Rivals_7](https://osu.ppy.sh/users/4610379) & [Voxa](https://osu.ppy.sh/users/9229539)
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open.
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com) and apply today!
+
+## <a id="mappers-showcase"></a>Mappers’ Showcase
+
+As we had all modes covered with the last issues of the Mappers’ Report it’s time again for the Storyboarder showcase! The unofficial 5th game mode of osu! gets way to less attention. We have many amazing storyboarders out there who are able to improve your playing experience by a lot. So let us jump right in!
```

Comment:
Let's not confuse ourselves with [Storyboarder](https://osu.ppy.sh/users/6124671), another prominent storyboarder
```suggestion
Because we had all modes covered with the last issues of the Mappers’ Report it’s time again for the storyboarder showcase! The unofficial 5th game mode of osu! gets way too little attention. We have many amazing storyboarders out there who are able to improve your playing experience by a lot. So let's jump right in!
```